src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;

```

Comment:
The fields 'color' and 'number' are properly encapsulated with private access modifiers. This is good practice as it adheres to the principle of encapsulation in OOP.